shared/src/vyxal/parsing/Parser.scala

```diff
@@ -84,6 +84,14 @@ object Parser:
         case TokenType.Command => parseCommand(token, asts, program) match
             case Right(ast) => asts.push(ast)
             case l => return l
+
+        case TokenType.NegatedCommand =>
+          parseCommand(token, asts, program) match
+            case Right(ast) =>
+              asts.push(ast)
```

Comment:
Doing that messes with literate to sbcs conversion. Pushing them individually is intended, because otherwise, the literate mode program no longer exactly matches the sbcs behaviour

Reply:
That's exactly what would happen and is intended behaviour. The idea is that it's just adding a ¬ anyway, so why should it be a single group

Reply:
`v<whatever>¬` is supposed to vectorise whatever and then negate. Consider it a quirk of literate mode

Reply:
An alternate idea would be to make ¬ arity group regardless of what comes before it